src/asmcnc/skavaUI/screen_rebooting.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, **kwargs):
     
     def on_pre_enter(self):
         self.reboot_label.text = self.l.get_str('Rebooting') + '...'
+        self.reboot_label.font_name = self.l.font_regular

```

Comment:
This is needed specifically for the situation that the user has just switched the software to Korean. As the screen has already been set up, when the language is changed to Korean, the font is still incorrect and will not display the text correctly when the user goes to reboot. As this is the only screen that the user should see after changing the language, the font is manually set on entry to display the Korean characters. (This is not required for the popup that tells the user to reboot, because it is created after the language switch, with the correct font)